packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -133,7 +143,8 @@ export function ModelsTree(props: ModelsTreeProps) {
                   {...rendererProps}
                   {...baseRendererProps}
                   nodeLoader={state.nodeLoader}
-                  nodeRenderer={(nodeProps) => <ModelsTreeNodeRenderer {...nodeProps} density={density} />}
+                  nodeRenderer={(nodeProps) => <ModelsTreeNodeRenderer {...nodeProps} density={density} reportUsage={reportUsage} />}
+                  reportUsage={reportUsage}
```

Comment:
Maybe `reportUsage` could be added to `nodeRenderer` props? So we wouldn't need to pass `reportUsage` to `FilterableTreeRenderer` and `ModelsTreeNodeRenderer`?